### PR TITLE
Deploying Safe contract in-app, ts cleanup, removed deprecated, scripts cleanup

### PR DESCRIPTION
**Pull request by pcowgill**
### Types of changes

Bug fix (non-breaking change that fixes an issue)

New feature (non-breaking change that adds functionality)

Meta (a change that affects future changes to this repo)

Technical debt (a code change that doesn't fix a bug or add a feature but makes something clearer for devs)

### CODE REVIEW

File: decentraland/helpers/index.ts [hunk 1]
```diff
@@ -155,25 +256,31 @@ export const fundAccountWithMana = (accountAddress): ActionObject => {
...
+const showToast = (msg: string): void =>
   Toast.show({ text: msg, duration: 3000, buttonText: "Okay" });
 
-export const showFatalError = (msg): void => console.error(msg);
-export const showError = (msg): void => showToast(`ERROR: ${msg}`);
-export const showWarn = (msg): void => showToast(`WARN: ${msg}`);
-export const showInfo = (msg): void => showToast(`${msg}`);
+export const showFatalError = (msg: string): void => console.error(msg);
+export const showError = (msg: string): void => showToast(`ERROR: ${msg}`);
+export const showWarn = (msg: any): void => showToast(`WARN: ${msg}`);
```
Comment (pedrobergamini) [inline decentraland/helpers/index.ts:278]: I think here msg could be annotated as string
```suggestion
export const showWarn = (msg: string): void => showToast(`WARN: ${msg}`);
```

File: decentraland/components/stateful/EthereumUpgradeSecurity/EthereumUpgradeSecurity.tsx [hunk 2]
```diff
@@ -2,38 +2,330 @@ import React, { useState, useEffect } from "react";
...
-    handleDeploymentComplete();
+    onCallSafeContractSubmit();
 
     return function cleanup() {
       // TODO: Cancel any unwanted remnants of having already made this request
     };
-  });
+  }, [account]); // Only re-run the effect if account changes
+
+  // TODO: Show loading by setting a boolean or enum value when deployment starts
```
Comment (pedrobergamini) [inline decentraland/components/stateful/EthereumUpgradeSecurity/EthereumUpgradeSecurity.tsx:293]: Maybe we could use a state hook for that to set loading value true/false inside `onCallSafeContractSubmit` try/catch block

**Comment by pedrobergamini** [comment]
Very nice! Only suggested a few small changes, what do you think?

File: decentraland/components/stateful/EthereumUpgradeSecurity/EthereumUpgradeSecurity.tsx (hunk 2 above)
Comment (pcowgill) [reply decentraland/components/stateful/EthereumUpgradeSecurity/EthereumUpgradeSecurity.tsx:293]: @pedrobergamini Yep, I think that sounds good!

Comment (pcowgill) [reply decentraland/components/stateful/EthereumUpgradeSecurity/EthereumUpgradeSecurity.tsx:293]: How would that handle persistence if someone quits the app and reopens it during the process?

Comment (pcowgill) [reply decentraland/components/stateful/EthereumUpgradeSecurity/EthereumUpgradeSecurity.tsx:293]: I created an issue to track this improvement. https://github.com/tasitlabs/tasit/issues/478

**Pull request merged**